src/sentry/tasks/summaries/daily_summary.py

```diff
@@ -0,0 +1,171 @@
+                    else:
+                        project_ctx.fourteen_day_total += total
+
+    with sentry_sdk.start_span(op="daily_summary.project_passes"):
+        for project in ctx.organization.project_set.all():
+            project_id = project.id
+            project_ctx = ctx.projects_context_map[project_id]
+            project_ctx.fourteen_day_avg = math.ceil(project_ctx.fourteen_day_total / 14)
+
+            # Today's Top 3 Error Issues
+            key_errors = project_key_errors(
+                ctx=ctx, project=project, referrer=Referrer.DAILY_SUMMARY_KEY_ERRORS.value
+            )
+            if key_errors:
+                project_ctx.key_errors = [(e["group_id"], e["count()"]) for e in key_errors]
+
+            # Today's Top 3 Performance Issues
+            key_performance_issues = project_key_performance_issues(
+                ctx=ctx,
+                project=project,
+                referrer=Referrer.DAILY_SUMMARY_KEY_PERFORMANCE_ISSUES.value,
+            )
+            if key_performance_issues:
+                project_ctx.key_performance_issues = key_performance_issues
+
+            # Issues that escalated or regressed today
+            regressed_or_escalated_groups = Group.objects.filter(
+                project=project, substatus__in=(GroupSubStatus.ESCALATING, GroupSubStatus.REGRESSED)
+            ).using_replica()
+            regressed_or_escalated_groups_today = Activity.objects.filter(
+                group__in=(regressed_or_escalated_groups),
+                type__in=(ActivityType.SET_REGRESSION.value, ActivityType.SET_ESCALATING.value),
+            )
+            if regressed_or_escalated_groups_today:
+                for activity in regressed_or_escalated_groups_today:
+                    if activity.type == ActivityType.SET_REGRESSION.value:
+                        project_ctx.regressed_today.append(activity.group)
+                    else:
+                        project_ctx.escalated_today.append(activity.group)
+
+            # The project's releases and the (max) top 3 new errors e.g. release - group1, group2
+            release_projects = ReleaseProject.objects.filter(project_id=project_id).values_list(
+                "release_id", flat=True
+            )
+            releases = Release.objects.filter(
+                id__in=release_projects, date_added__gte=ctx.end.date()
+            )
+            for release in releases[:2]:  # or whatever we limit this to
+                new_groups_in_release = Group.objects.filter(project=project, first_release=release)
+                if new_groups_in_release:
+                    project_ctx.new_in_release = {
+                        release.id: [group for group in new_groups_in_release]
+                    }
+    with sentry_sdk.start_span(op="weekly_reports.fetch_key_error_groups"):
+        fetch_key_error_groups(ctx)
+
+    with sentry_sdk.start_span(op="weekly_reports.fetch_key_performance_issue_groups"):
```

Comment:
should these spans be prefixed with `daily_summary` instead of `weekly_reports`?